public/style.css

```
@@ -1,1003 +1,1106 @@
+    gap: 0.5rem;
+}
+
+.social-links a:hover {
+    color: #667eea;
+    border-color: #667eea;
+    transform: translateY(-3px);
+    box-shadow: 0 5px 15px rgba(102, 126, 234, 0.2);
+}
+
+/* Responsive Design */
+@media (max-width: 768px) {
```

Stylelint (17.11.1)

[error] Unclosed block (CssSyntaxError)

(CssSyntaxError)